sporkfish/searcher/minimax.py

```diff
@@ -195,6 +202,13 @@ def _quiescence(self, board: Board, depth: int, alpha: float, beta: float) -> fl
         :rtype: float
         """
 
+        # Probe the transposition table for an existing entry
+        # We treat all cases as depth 0, so essentially as an static evaluation
```

Comment:
Sorry can you elaborate this inline comment?

Reply:
This potentially is abit dodgy but I see OK performance and moves aren't that bad. Essentially TT retrieves moves with higher depth than previous moves. By treating all quiescence moves at depth 0, if I encounter the position again I essentially skip having it being evaluated regardless of the quiescence depth. This is in contrast to taking the depth to go from quiescence depth 4 to 0 which is essentially 0 to -4 in our negamax depth.

However with how it's evaluated the score is recursively propagated back up so I didn't see the issue in doing it - the "negative depth" score gets evaluated first and then propagated back up to depth 0, so technically if the depth 0 position gets encountered again it's still the same score.

Reply:
Hmm interestinggg - my brain is a bit fried atm, need to thinking deeper about this one later

But agree, seems to be working as the tests are not breaking